runtime/lua/vscode-neovim.lua

```diff
@@ -1,30 +1,28 @@
 local api = require("vscode-neovim.api")
-local defaults = require("vscode-neovim.defaults")
-local cursor = require("vscode-neovim.cursor")
-
-require("vscode-neovim.highlight")
-require("vscode-neovim.filetype")
 
-local M = {}
+local default_optons = require("vscode-neovim.default-options")
+local cursor = require("vscode-neovim.cursor")
+local highlight = require("vscode-neovim.highlight")
+local filetype = require("vscode-neovim.filetype")
+local numbers = require("vscode-neovim.numbers")
+local autocmds = require("vscode-neovim.autocmds")
 
-M.notify = api.notify
-M.call = api.call
-M.call_range = api.call_range
-M.notify_range = api.notify_range
-M.call_range_pos = api.call_range_pos
-M.notify_range_pos = api.notify_range_pos
+default_optons.setup()
```

Comment:
We still need to expose setup as part of this module. We don't want to call setup every time vscode-neovim is required. Setup should be once on init, like it was before.

Reply:
The setup needs to be reserved for users to configure features.

Lua will not reload an already loaded module unless the user manually modifies `package.loaded`...

Reply:
Improved